local-requirements.txt

```diff
@@ -7,7 +7,7 @@ mypy==1.11.2
 objgraph==3.6.1
 Pillow==10.4.0
 pixelmatch==0.3.0
-pre-commit==3.4.0
+pre-commit==3.5.0
```

Comment:
What is this one? Why is it different from the previous file?

Reply:
This is `pre-commit` - like ESLint - and above are some rules we consume.